model/src/main/scala/za/co/absa/atum/model/types/basic.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.model.types
+
+import scala.collection.immutable.ListMap
+import za.co.absa.atum.model.dto.{PartitionDTO, PartitioningDTO}
+
+
+object basic {
```

Comment:
Good idea to move this into its dedicated object instead of the AtumContext. The Ops are also useful.